lib/charms/opensearch/v0/opensearch_tls.py

```diff
@@ -302,6 +302,7 @@ def _request_certificate_renewal(
         subject = self._get_subject(cert_type)
         organization = self.charm.opensearch_peer_cm.deployment_desc().config.cluster_name
         new_csr = generate_csr(
+            add_unique_id_to_subject_name=False,
```

Comment:
Wow, nice catch!!!!!

(Perhaps an explanatory comment above as well, why this is essential?)

Reply:
Double-checking: why did this conversation get closed? 

Decision on no comment or perhaps by accident?
(I think it may be nice for the next developer in 6 months or so to have a quick reminder why this boolean was critical. Regardless, I leave it to your judgement :-) )

Reply:
Hey Judit, yeah I decided not to comment, as the change is just to use the exactly same parameters as in the other `generate_csr` call in the code. To me this was just a bug I fixed along the way, nothing worth to comment.